// compile_and_run_and_check_exit_status 2

func main() -> int { return 2; }
func foo(a: Foo) { }
class Foo { const i: int; }
